region/crateria/east.json

```diff
@@ -1772,8 +1772,51 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": [],
-                  "note": "It's horribly time-consuming, but this can be attained without any items or techs."
+                  "requires": [ "h_canFly" ]
+                },
+                {
+                  "name": "Climb the Trippers with HiJump",
+                  "notable": false,
+                  "requires": [
+                    "canUseEnemies",
+                    "HiJump"
+                  ]
+                },
+                {
+                  "name": "Climb the Trippers Bootless",
+                  "notable": false,
+                  "requires": [
+                    "canUseEnemies",
+                    {"or": [
+                      "canCarefulJump",
+                      "canUseFrozenEnemies"
+                    ]},
+                    {"or": [
```

Comment:
I think this is to jump into the item.  But you don't need any extra height gain to reach it?

Reply:
These are ways to climb the platforms. I would say jumping from the middle platform to the item would probably be a `tricky jump`, because the way you will likely encounter it has the top tripper in the way and requires a somewhat precise shot and then somewhat precise jump, where a failure will surely waste a lot of time.

Reply:
The tricky jump strat could be added, but I think the careful + cjdg is more intuitive and easier.